src/safe_apps/migrations/0013_alter_safeapp_icon_url.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 5.0.6 on 2024-05-31 14:46
+# Modified to update existing entries with no icon_url
+
+from django.db import migrations, models
+from django.db.migrations.state import StateApps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+import safe_apps.models
+
+
+def add_missing_default_icon_url(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    SafeApp = apps.get_model("safe_apps", "SafeApp")
+    default_icon_url = "safe_apps/icon_url.jpg"
+
+    SafeApp.objects.filter(icon_url__isnull=True).update(icon_url=default_icon_url)
+    SafeApp.objects.filter(icon_url="").update(icon_url=default_icon_url)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("safe_apps", "0012_rename_visible_safeapp_listed"),
+    ]
+
+    operations = [
+        # Changing field from nullable to non-nullable is impossible in the same transaction
+        migrations.RunSQL(
+            "SET CONSTRAINTS ALL IMMEDIATE", reverse_sql="SET CONSTRAINTS ALL DEFERRED"
```

Comment:
Why do you need this? It shouldn't be required

Reply:
As discussed, as setting a value and changing the field to non-nullable is otherwise not possible in the same migration, these two operations have been separated into separate migrations in e2c43374aeb23e17d742a5fca7e24cd3f85fa36b.